app/(connected)/(with-sidebar)/(without-chat)/mes-jeunes/[jeune_id]/actions/nouvelle-action/page.tsx

```diff
@@ -0,0 +1,49 @@
+import { Metadata } from 'next'
+import React from 'react'
+
+import NouvelleActionPage, {
+  TITRE_AUTRE,
+} from 'app/(connected)/(with-sidebar)/(without-chat)/mes-jeunes/[jeune_id]/actions/nouvelle-action/NouvelleActionPage'
+import {
+  PageHeaderPortal,
+  PageRetourPortal,
+} from 'components/PageNavigationPortals'
+import { getSituationsNonProfessionnelles } from 'services/actions.service'
+import { getActionsPredefinies } from 'services/referentiel.service'
+import { getMandatorySessionServerSide } from 'utils/auth/auth'
+
+export const metadata: Metadata = {
+  title: 'Créer action - Actions jeune',
```

Comment:
```suggestion
  title: 'Créer une nouvelle action - Actions jeune',
```
pour le jeune comme on part de la fiche bénéficiaire, il s'agit d'une action le concernant, donc reprendre dans le title le nom prénom?

Reply:
C'est relou parce que faut du coup récupérer le jeune dans la construction des metadata.

Reply:
oui mais si tu as plusieurs onglets d'ouverts, le title n'est pas unique alors que ça peut être une page nouvelle action pour des jeunes différents :-/